test/unit/specs/composables/use-i18n-utilities-in-component.spec.js

```diff
@@ -0,0 +1,42 @@
+import VueI18n from 'vue-i18n'
+import { render, screen } from '@testing-library/vue'
+import { AUDIO, IMAGE } from '~/constants/media'
+import VContentLink from '~/components/VContentLink/VContentLink.vue'
+
+const enMessages = require('~/locales/en.json')
+const i18n = new VueI18n({
+  locale: 'en',
+  fallbackLocale: 'en',
+  messages: { en: enMessages },
+})
+
+describe('i18nResultsCount in a component', () => {
+  let options = {}
+  let count
+  beforeEach(() => {
+    options = {
+      props: { mediaType: 'image', resultsCount: count },
+      mocks: { $nuxt: { context: { i18n } } },
+    }
+  })
+
+  it.each`
+    resultCount | mediaType | expectedText
+    ${0}        | ${IMAGE}  | ${'No image results'}
+    ${0}        | ${AUDIO}  | ${'No audio results'}
+    ${10}       | ${IMAGE}  | ${'10 image results'}
+    ${10}       | ${AUDIO}  | ${'10 audio results'}
```

Comment:
Should we add `1` here as well given it usually creates a distinct branch for plurals in English?

```suggestion
    ${0}        | ${IMAGE}  | ${'No image results'}
    ${0}        | ${AUDIO}  | ${'No audio results'}
    ${1}        | ${IMAGE}  | ${'1 image result'}
    $10}        | ${AUDIO}  | ${'1 audio result'}
    ${10}       | ${IMAGE}  | ${'10 image results'}
    ${10}       | ${AUDIO}  | ${'10 audio results'}
```